packages/create-cloudflare/src/pages.ts

```diff
@@ -174,3 +179,27 @@ const createProject = async (ctx: PagesGeneratorContext) => {
 		crash("Failed to create pages application. See output above.");
 	}
 };
+
+const generatePagesCommitMessage = (ctx: PagesGeneratorContext) => {
+	const header = "Web application initialized by Create-Cloudflare CLI";
+
+	const details: { key: string; value: string }[] = [
+		{ key: "date", value: new Date().toISOString() },
+		{ key: "create-cloudflare version", value: version },
+		{ key: "project name", value: ctx.project.name },
+	];
+
+	if (ctx.framework) {
+		details.push({ key: "framework", value: `${ctx.framework.name}` });
+		details.push({
+			key: "framework cli version",
+			value: `${getFrameworkVersion(ctx)}`,
```

Comment:
we can specify the cli version since we don't really know if the framework version is the same

I could add some code to also check the generated package.json and find the framework's version in that way if we want, but that feels pretty brittle to me and would require some mapping (for example for qwik I'd have to map qwik to `@builder.io/qwik`), so unless we really think that the framework's version is important I'd avoid that 🤔

Reply:
I think the `create-*` cli version here is fine. It's slightly annoying but totally possible to reverse lookup. If we're hitting issues this version is more valuable from a diagnostic standpoint.